doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -1237,6 +1270,35 @@ Check using ``bps report --id ID``.
 
 If that's the case, the jobs can often be edited and released in a plugin-specific way.
 
+.. _bps_job_running_again:
+
+Why is my job running again?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, BPS will ask the WMS to retry jobs :ref:`numberOfRetries <bps_number_of_retries>` times
+except if the job exits with an exit code in :ref:`retryUnlessExit <bps_retry_unless_exit>`.
+See the ``*_config.yaml`` file in the submit directory for the default values.  If there is an exit code
+that should be added to the default value, create a JIRA ticket requesting the change.
+
+.. _bps_why_not_retrying_job:
+
+Why isn't the WMS retrying my job that failed?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, BPS will ask the WMS to retry jobs :ref:`numberOfRetries <bps_number_of_retries>` times
+except if the job exits with an exit code in :ref:`retryUnlessExit <bps_retry_unless_exit>`.
+
+Either:
+
+* the WMS doesn't support retrying jobs at all.  Check WMS-specific documentation.
+
+* numberOfRetries is 0 (or None).  See the ``*_config.yaml`` file in the submit
```

Comment:
In the description of the `numberOfRetries` above and in the examples that follow, the documentation instructs the user explicitly to use the value `null` to disable automatic retries. So I' assume that it can't be set to 0 or `None` in the config file. Or can it?

Reply:
Tests with 0 worked.  null in yaml is translated to None in python and that works.  None in the yaml is treated as the string "None" and is put in the condor submit file and then condor complains: `submit error: Submit:-1:max_retries=None is invalid, must eval to an integer.`  While null works, I will just document the 0 for this particular value since the name `numberOfRetries` implies numeric value.